docs/src/lib/operations.md

```diff
@@ -223,9 +223,7 @@ Inherited from [`LazySet`](@ref):
 
 ```@docs
 LinearMap
-*(::AbstractMatrix{N}, ::LazySet{N}) where {N<:Real}
-*(::N, ::LazySet{N}) where {N<:Real}
-*(::N, ::LinearMap{N}) where {N<:Real}
+*(::Any, ::LazySet)
```

Comment:
I think we shouldn't be using `Any` for this operation. It is used for matrices, numeric types or sets.